cli/cmd/container_registry.go

```diff
@@ -0,0 +1,194 @@
+package cmd
+
+import (
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/lacework/go-sdk/api"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	containerRegistryCommand = &cobra.Command{
+		Use:     "container-registry",
+		Aliases: []string{"container-registries", "cr"},
+		Short:   "Manage container registries",
+		Long:    "Manage container registries integrations with Lacework",
+	}
+
+	// containerRegistriesListCmd represents the list sub-command inside the container registries command
+	containerRegistryListCmd = &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List all available container registry integrations",
+		Args:    cobra.NoArgs,
+		RunE:    containerRegistryList,
+	}
+
+	// containerRegistryShowCmd represents the show sub-command inside the container registry command
+	containerRegistryShowCmd = &cobra.Command{
+		Use:     "show",
+		Aliases: []string{"get"},
+		Short:   "Show a single container registry integration",
+		Args:    cobra.ExactArgs(1),
+		RunE:    containerRegistryShow,
+	}
+
+	// containerRegistryCreateCmd represents the show sub-command inside the container registries command
+	containerRegistryCreateCmd = &cobra.Command{
+		Use:   "create",
+		Short: "Create a new container registry integration",
+		Args:  cobra.NoArgs,
+		RunE:  containerRegistryCreate,
+	}
+
+	// containerRegistryDeleteCmd represents the delete sub-command inside the container registries command
+	containerRegistryDeleteCmd = &cobra.Command{
+		Use:     "delete",
+		Aliases: []string{"rm"},
+		Short:   "Delete a container registry integration",
+		Args:    cobra.ExactArgs(1),
+		RunE:    containerRegistryDelete,
+	}
+)
+
+func init() {
+	// add the container-registry command
+	rootCmd.AddCommand(containerRegistryCommand)
+	containerRegistryCommand.AddCommand(containerRegistryListCmd)
+	containerRegistryCommand.AddCommand(containerRegistryShowCmd)
+	containerRegistryCommand.AddCommand(containerRegistryCreateCmd)
+	containerRegistryCommand.AddCommand(containerRegistryDeleteCmd)
+}
+
+func containerRegistriesToTable(integrations []api.ContainerRegistryRaw) [][]string {
+	var out [][]string
+	for _, cadata := range integrations {
+		out = append(out, []string{
+			cadata.IntgGuid,
+			cadata.Name,
+			cadata.Type,
+			cadata.Status(),
+			cadata.StateString(),
+		})
+	}
+	return out
+}
+
+func containerRegistryList(_ *cobra.Command, _ []string) error {
+	containerRegistries, err := cli.LwApi.V2.ContainerRegistries.List()
+
+	if err != nil {
+		return errors.Wrap(err, "unable to get container registries")
+	}
+
+	if len(containerRegistries.Data) == 0 {
+		cli.OutputHuman("No container registries found.\n")
+		return nil
+	}
+
+	if cli.JSONOutput() {
+		return cli.OutputJSON(containerRegistries.Data)
+	}
```

Comment:
Order here matters, if users wants JSON format, we should NOT print human readable. Please update any other place where this orders is wrong:
```suggestion
	if cli.JSONOutput() {
		return cli.OutputJSON(containerRegistries.Data)
	}
	
	if len(containerRegistries.Data) == 0 {
		cli.OutputHuman("No container registries found.\n")
		return nil
	}
```